packages/server/utils/authorization.ts

```diff
@@ -1,17 +1,27 @@
 import type AuthToken from '../database/types/AuthToken'
 import type {DataLoaderWorker} from '../graphql/graphql'
 import type {OrganizationUser} from '../postgres/types'
+import logError from './logError'
 
-export const getUserId = (authToken: any) => {
+const getUserIdInternal = (authToken: any) => {
   return authToken && typeof authToken === 'object' ? (authToken.sub as string) : ''
 }
 
+export const getUserId = (authToken: any) => {
+  const userId = getUserIdInternal(authToken)
+  if (userId === 'aGhostUser') {
+    // This probably means someone is trying to get the viewerId when the mutation was called from the server
+    logError(new Error('getUserId called with aGhostUser'))
```

Comment:
Logging this as error to avoid confusing the server token with the one from the viewer